Mathlib/RingTheory/Polynomial/Hermite/Basic.lean

```diff
@@ -172,7 +172,7 @@ theorem coeff_hermite_explicit :
       simp only
       -- Factor out (-1)'s.
       rw [mul_comm (↑k + _ : ℤ), sub_eq_add_neg]
-      nth_rw 3 [neg_eq_neg_one_mul]
+      nth_rw 4 [neg_eq_neg_one_mul]
```

Comment:
how concerning is this?

Reply:
It's wrong, and doesn't work, so I'm reverting it now, but there remains the mystery of why it has to be `4` and not `3` on nightly-testing!

Reply:
Ah, the proofs diverge later, and then one on nightly-testing is worse. Sorry, this was just noise from the complicated merges. I've cleanup up `nightly-testing` now so we won't get this diff showing up again.